React/wbooks/src/SignUp/layout.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Field, reduxForm } from 'redux-form';
+
+import './style.css';
+
+const required = value => (value ? undefined : 'Requerido');
```

Comment:
I'm pretty sure you have the same or almost the same validations here than in logout, so extract to inputValidations.js in a utils folder below `src`